src/components/transport_manager/test/tcp_transport_adapter_test.cc

```diff
@@ -365,7 +365,7 @@ TEST_F(TcpAdapterTest, NotifyTransportConfigUpdated) {
 }
```

Comment:
```
  EXPECT_CALL(mock_adapter_listener, OnTransportConfigUpdated(_)).Times(1);
```

Specify expected parameter

Reply:
@alexkutsan 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/3134/commits/5d3cb26512471af4b74f4c069d488f8916779c3e